### PR TITLE
Feature/bpool activation fixup

### PR DESCRIPTION
**Pull request by dchvs**

### CODE REVIEW

File: gst-libs/gst/tiovx/gsttiovxpad.c [hunk 1]
```diff
@@ -406,6 +406,10 @@ gst_tiovx_pad_acquire_buffer (GstTIOVXPad * self, GstBuffer ** buffer,
 
   priv = gst_tiovx_pad_get_instance_private (self);
 
+  if (!gst_buffer_pool_is_active (GST_BUFFER_POOL (priv->buffer_pool))) {
+    gst_buffer_pool_set_active (GST_BUFFER_POOL (priv->buffer_pool), TRUE);
+  }
```
Comment (emmadrigal) [inline gst-libs/gst/tiovx/gsttiovxpad.c:411]: Can you leave this as?
```
     gst_buffer_pool_set_active (GST_BUFFER_POOL (priv->buffer_pool), TRUE); 
```

File: gst-libs/gst/tiovx/gsttiovxutils.c [hunk 2]
```diff
@@ -570,6 +572,10 @@ gst_tiovx_validate_tiovx_buffer (GstDebugCategory * category,
       return FALSE;
     }
 
+    if (!gst_buffer_pool_is_active (GST_BUFFER_POOL (new_pool))) {
+      gst_buffer_pool_set_active (GST_BUFFER_POOL (new_pool), TRUE);
+    }
```
Comment (emmadrigal) [inline gst-libs/gst/tiovx/gsttiovxutils.c:577]: Can you leave this as?

     gst_buffer_pool_set_active (GST_BUFFER_POOL (priv->buffer_pool), TRUE);

**Review by emmadrigal — CHANGES_REQUESTED** [review]
Have you tested this with MISO modules?

File: gst-libs/gst/tiovx/gsttiovxutils.c [hunk 3]
```diff
@@ -405,6 +404,9 @@ gst_tiovx_buffer_copy (GstDebugCategory * category, GstBufferPool * pool,
   g_return_val_if_fail (pool, NULL);
   g_return_val_if_fail (in_buffer, NULL);
 
+  /* Activate the buffer pool */
+  gst_buffer_pool_set_active (GST_BUFFER_POOL (pool), TRUE);
+
```
Comment (michaelgruner) [inline gst-libs/gst/tiovx/gsttiovxutils.c:409]: This one feels unnecessary, acquire already activates the pool.

**Review by michaelgruner — CHANGES_REQUESTED** [review]

File: gst-libs/gst/tiovx/gsttiovxutils.c (hunk 3 above)
Comment (dchvs) [reply gst-libs/gst/tiovx/gsttiovxutils.c:409]: This activation satisfies the cases for non TIOVX buffers.

**Pull request merged**